realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -1143,6 +1143,50 @@ public void copyToRealm_cyclicObjectReferences() {
         assertEquals("One", realmObject.getName());
         assertEquals("Two", realmObject.getObject().getName());
         assertEquals(2, realm.where(CyclicType.class).count());
+
+        realm.beginTransaction();
```

Comment:
Either split this into a another unit test or add a comment about what is now being tested.